### ISSUE TITLE
Non rigid deformation between a template and target mesh with different vertices

### ISSUE THREAD

**Issue by saikiran321** [tensorflow/graphics#446]
This is an amazing library, I was wonder if is possible to model  As Conformal As Possible fitting energy between two meshes that have completely different vertex and edge information with this library?

Example:

mesh_1 contains   10000 vertices and 25000 faces
mesh_2 contains  2000 vertices and 10000 faces, in this case, how would we proceed ?

**Comment by topinfrassi01** [comment]
The Chamfer loss function doesn't require the two meshes to have the same number of vertices, so you could use that as a start.

In general, Tensorflow doesn't care if the predictions and the ground truth don't have the same shape, but most losses expect it.
